dataframe-jupyter/src/main/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/JupyterHtmlRenderer.kt

```diff
@@ -81,10 +74,10 @@ internal inline fun <reified T : Any> JupyterHtmlRenderer.render(
         val jsonEncodedDf = when {
             !ideBuildNumber.supportsDynamicNestedTables() -> {
                 buildJsonObject {
-                    put(NROW, df.size.nrow)
-                    put(NCOL, df.size.ncol)
-                    putJsonArray(COLUMNS) { addAll(df.columnNames()) }
-                    put(KOTLIN_DATAFRAME, encodeFrame(df.take(limit)))
+                    put(SerializationKeys.NROW, df.rowsCount())
```

Comment:
Looks like these few internal declarations can be hidden behing single `buildJsonObject(df, limit)`